### DISCUSSION TITLE
Account Creation Design

### DISCUSSION THREAD

**Discussion by noel-vargas** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#60] [Ideas]
I currently have this draft for the account creation and Login UI design, would love some feedback.
![image](https://github.com/user-attachments/assets/d1c4a2b1-8708-433b-a286-c97ff145e642)

![image](https://github.com/user-attachments/assets/8aae99bf-f7cb-471b-982f-587148bf8e92)

**Comment by yadrielsamir** [comment]
The overall inputs are fitting but the design can be a bit more modern. I suggest centralizing the inputs for the create account page and changing the green buttons to a darker green. Additionally, some decorative aspects can be added to the pages design. The design I uploaded for admins profile in a previous discussion had font Cairo and 0F8F46 color, just in case.
